examples/hikari_lightbulb_basic_queue/main.py

```diff
@@ -0,0 +1,38 @@
+import os
+
+import hikari
+import lightbulb
+import lavasnek_rs
+
+PREFIX = "!" # Should probably set this to an environment variable
```

Comment:
Be consistent with the other examples.
```suggestion
PREFIX = "," # Should probably set this to an environment variable
```